packages/@okta/vuepress-site/docs/guides/custom-widget/main/index.md

```diff
@@ -16,70 +16,81 @@ This guide explains how to customize the Sign-In Widget when Okta is hosting it
 **What you need**
 
 * [Okta Developer Edition organization](https://developer.okta.com/signup)
-* (Okta-hosted) [Custom domain](/docs/guides/custom-url-domain/main/)
-* (Self-hosted) [Okta Sign-In Widget](https://github.com/okta/okta-signin-widget#embedded-self-hosted) installed into your project with configured authentication scenarios
+* (Redirect authentication): [Custom URL domain](/docs/guides/custom-url-domain/main/)
```

Comment:
why is (Redirect authentication) using parenthesis and below not?

Reply:
fixed